index.html

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
you can add meta tag for author , although it is not mandatory, just a suggestion

Reply:
Similar suggestion. This increases the accessibility of your website and helps in SEO.